### ISSUE TITLE
Content: Add Home pg "pitch deck" introduction

### ISSUE THREAD

**Issue by thansidwell** [CodeForPhilly/vacant-lots-proj#338]
The existing home doesn't introduce people to the whole concept of Clean & Green Philly. That happens on the About page. We want to change that and move the "pitch deck" for Clean & Green Philly right on the home page. 

- Here's the Figma prototype. https://www.figma.com/file/NAFkgq34abW6uJ0R7PW24T/Prototype---Clean-%26-Green-Philly?type=design&node-id=2266%3A23177&mode=design&t=jX4y1auMbn2sVoGD-1
- Add these 8 new sections to the homepage including
  - Heading
  - Body text
  - Button
  - Image
- Here's the images exported. [Home Page Images.zip](https://github.com/CodeForPhilly/vacant-lots-proj/files/14240688/Home.Page.Images.zip)

![Image](https://github.com/CodeForPhilly/vacant-lots-proj/assets/1965986/5ba43b91-6a35-4ddf-9d2d-8e11b976e30f)

### Dev Notes:

- Some headings are need to be numbered and indented. https://share.cleanshot.com/1WrldLSD I looked up if that was possible and found the start attribute. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol#using_the_start_attribute
```
<h2 class="heading-2xl">
  <ol start="2">
    <li>Get data-driven suggestions on how to legally get access.</li>
  </ol>
</h2>
```

**Comment by ajtw7** [comment]
Hey @nlebovits, you can assign this to me as well.

**Comment by bacitracin** [comment]
Random A11y comment - The numbered headings are all in order and I think it makes sense to group them as one list under "We created the ultimate toolkit..." so it might look something like this. 

```
<h2>We created the ultimate toolkit... </h2>
<ol>
<li>
<h3>Find Vacant properties that match your goals</h3>
<p></p>
</li>
<li>
<h3>Get Data Driven Suggestions</h3>
<p></p>
</li>
</ol>
<h3></h3>
```

**Comment by bacitracin** [comment]
@marvieqa  do you think the ordered list items would be grouped under that h2 "We created the ultimate toolkit"? Visually it looks equal, but the contents are basically what the blue 1/2/3 images are mapping out.

**Comment by marvieqa** [comment]
@bacitracin 💯 agree on preserving the meaning and the sample code of making the ordered list as `<h3>` under the `<h2>"We created the ultimate toolkit..."</h2>`.  I tested this flow and the succeeding `<p>`s aren't automatically announced.

**Comment by ajtw7** [comment]
Hey @thansidwell — I've asked @nlebovits to un-assign me from this task. I think @bacitracin or @marvieqa might have the bandwidth for it..?
I'm honestly not sure who's working on what task, regardless of who appears to be assigned to it on the backlog. 
And I wouldn't want to write code for an issue that's been fixed.

**Comment by Moylena** [comment]
@paulhchoi  if you have time - this ticket is our #1 front end priority. If you want to chunk it out by section I'm happy to break this ticket down into further components.

**Comment by paulhchoi** [comment]
> @paulhchoi if you have time - this ticket is our #1 front end priority. If you want to chunk it out by section I'm happy to break this ticket down into further components.

Sounds good, I can take this on. 

@thansidwell the zip file with the images in the description is 404ing for me, could you reshare the images assets please?

**Comment by paulhchoi** [comment]
@thansidwell I see the following placeholder in the figma prototype, could you inform on how I should dev these sections?

<img width="504" alt="image" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/8061917/15a105e1-a1c2-44e9-879d-a46705f7a0a5">

**Comment by marvieqa** [comment]
@paulhchoi 👋 We made a11y annotations in the design as well that might be useful. I'm attaching the screenshot below because I think not everyone can show/hide the layers in Figma. cc @bacitracin 

<details>
<summary> A11y annotation screenshots </summary>

![Screenshot 2024-03-13 at 13 36 15](https://github.com/CodeForPhilly/vacant-lots-proj/assets/102739972/798cc21d-09a1-4cb2-b743-8780f3098e7c)

![Screenshot 2024-03-13 at 13 36 38](https://github.com/CodeForPhilly/vacant-lots-proj/assets/102739972/bf07ea61-3c88-41f7-a118-9bb846140df4)

![Screenshot 2024-03-13 at 13 42 38](https://github.com/CodeForPhilly/vacant-lots-proj/assets/102739972/73771f4d-d2fa-419b-84e0-9d307e74d63c)

</details>

**Comment by paulhchoi** [comment]
@marvieqa this was helpful, thanks! Just pushed a commit into the draft PR with the spec you provided. I may need some guidance on how to properly structure the `<ol>` of the three numbered 'slides' — I contained all three in an `<ol>`, then made each section its own `<li>`, but in order to have the headers number and align the way we wanted to, I had to wrap each header in their own `<ol>` and `<li>`. Given nested lists and lighthouse's greenlight, I think this should be sufficient, but open to feedback
